### DISCUSSION TITLE
How to modify the active/inactive pane color scheme?

### DISCUSSION THREAD

**Discussion by windmgc** [wezterm/wezterm#2288] [Q&A] [answered]
I want to make the contrast ratio more obvious between the active and inactive panes. It seems that by default the active pane will be a bit more bright than the inactive ones. How can I achieve that?

**Comment by MuhammedZakir** [comment, marked as answer]
https://github.com/wez/wezterm/discussions/2285
https://wezfurlong.org/wezterm/config/appearance.html#styling-inactive-panes

**Reply by windmgc** [reply]
Sorry, it seems that I totally missed that part...Thanks so much for your blazing fast response!
